build/cross-images/x86_64-unknown-linux-musl.dockerfile

```diff
@@ -11,3 +11,5 @@ RUN echo "Upgrading CMake" && \
 ENV CC=musl-gcc \
     CFLAGS=-I/usr/local/musl/include \
     LIB_LDFLAGS=-L/usr/lib/x86_64-linux-gnu
+
+RUN sudo chown -R rust:rust /opt/rust/
```

Comment:
Just curious, why is this needed? Also, do you need `sudo`, I believe docker containers are built and run using the user `root`.

Reply:
No worries :+1:

Reply:
On the other hand for the dependencies yes, RocksDB need LLVM and Clang to work properly